### ISSUE TITLE
Error: void-function org-link-set-parameters 

### ISSUE THREAD

**Issue by seagle0128** [skeeto/elfeed#301]
- macOS Mojave
- GNU Emacs 25.1
- org 9.2
- elfeed-20181127.1943

```elisp
Debugger entered--Lisp error: (void-function org-link-set-parameters)
  (org-link-set-parameters "elfeed" :follow (function elfeed-link-open) :store (function elfeed-link-store-link))
  (with-no-warnings (org-link-set-parameters "elfeed" :follow (function elfeed-link-open) :store (function elfeed-link-store-link)))
  (if (version< (org-version) "9.0") (with-no-warnings (org-add-link-type "elfeed" (function elfeed-link-open)) (add-hook (quote org-store-link-functions) (function elfeed-link-store-link))) (with-no-warnings (org-link-set-parameters "elfeed" :follow (function elfeed-link-open) :store (function elfeed-link-store-link))))
  (lambda nil (if (version< (org-version) "9.0") (with-no-warnings (org-add-link-type "elfeed" (function elfeed-link-open)) (add-hook (quote org-store-link-functions) (function elfeed-link-store-link))) (with-no-warnings (org-link-set-parameters "elfeed" :follow (function elfeed-link-open) :store (function elfeed-link-store-link)))))()
  funcall((lambda nil (if (version< (org-version) "9.0") (with-no-warnings (org-add-link-type "elfeed" (function elfeed-link-open)) (add-hook (quote org-store-link-functions) (function elfeed-link-store-link))) (with-no-warnings (org-link-set-parameters "elfeed" :follow (function elfeed-link-open) :store (function elfeed-link-store-link))))))
  (lambda nil (funcall (quote (lambda nil (if (version< (org-version) "9.0") (with-no-warnings (org-add-link-type "elfeed" (function elfeed-link-open)) (add-hook (quote org-store-link-functions) (function elfeed-link-store-link))) (with-no-warnings (org-link-set-parameters "elfeed" :follow (function elfeed-link-open) :store (function elfeed-link-store-link))))))))()
  eval-after-load(org (funcall (quote (lambda nil (if (version< (org-version) "9.0") (with-no-warnings (org-add-link-type "elfeed" (function elfeed-link-open)) (add-hook (quote org-store-link-functions) (function elfeed-link-store-link))) (with-no-warnings (org-link-set-parameters "elfeed" :follow (function elfeed-link-open) :store (function elfeed-link-store-link))))))))
  eval-buffer(#<buffer  *load*-233425> nil "/home/travis/.emacs.d/elpa/elfeed-20181127.1943/elfeed-autoloads.el" nil t)  ; Reading at buffer position 1774
  load-with-code-conversion("/home/travis/.emacs.d/elpa/elfeed-20181127.1943/elfeed-autoloads.el" "/home/travis/.emacs.d/elpa/elfeed-20181127.1943/elfeed-autoloads.el" nil t)
  load("/home/travis/.emacs.d/elpa/elfeed-20181127.1943/elfeed-autoloads" nil t)
  package--activate-autoloads-and-load-path([cl-struct-package-desc elfeed (20181127 1943) "an Emacs Atom/RSS feed reader" ((emacs (24 3))) nil nil "/home/travis/.emacs.d/elpa/elfeed-20181127.1943" nil nil])
  package--load-files-for-activation([cl-struct-package-desc elfeed (20181127 1943) "an Emacs Atom/RSS feed reader" ((emacs (24 3))) nil nil "/home/travis/.emacs.d/elpa/elfeed-20181127.1943" nil nil] :reload)
  package-activate-1([cl-struct-package-desc elfeed (20181127 1943) "an Emacs Atom/RSS feed reader" ((emacs (24 3))) nil nil "/home/travis/.emacs.d/elpa/elfeed-20181127.1943" nil nil] :reload :deps)
```

**Comment by fleimgruber** [comment]
Maybe the problem is that the org-mode 8.x shipped with Emacs 25.1 is loaded and not org 9.x? See also e.g. https://github.com/jkitchin/org-ref/issues/561. AFAICS, https://github.com/seagle0128/.emacs.d/commit/605b86a52476a9c1c83a7944f67d453412536214 is a only a guard for your CI and not really a fix? In any case, I think this is not an elfeed issue, so please consider closing.

**Comment by seagle0128** [comment]
This issue is same as https://github.com/jkitchin/org-ref/issues/561. It only exists in fresh startup, and disappears after restarting. That's related to the org version and loading sequence of the packages.

Anyway I will only load it with org 9.0+. Close it.
